smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/ErrorHandlerIT.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Adobe Systems Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.cq.cloud.testing.it.smoke;
+
+import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.junit.rules.CQAuthorPublishClassRule;
+import org.apache.sling.testing.clients.ClientException;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+/**
+ * This test class is protecting from flawed AEM error handler implementations that are not returning 404s in case
+ * of page not found.
+ */
+public class ErrorHandlerIT {
```

Comment:
The class name is visible in many places, it would be good to have a more specific and self-explanatory name. Maybe: `PageNotFoundHandlerIT`? 

EDIT: I get it's structured to cover the Error Handler structure in AEM, so no strong position on this.